confluent_kafka/avro/cached_schema_registry_client.py

```diff
@@ -269,7 +270,8 @@ def test_compatibility(self, subject, avro_schema, version='latest'):
             else:
                 log.error("Unable to check the compatibility")
                 False
-        except:
+        except: # noqa
+            log.error("_send_request() failed: %s" % sys.exc_info()[0])
```

Comment:
Best practice is to use the `logging` framework's built-in string interpolation:

    log.level("string %s", arg1)

That way the string isn't built unless the log level is hit. For error-level, this is simply a style thing, but for debug-level logging this can occasionally have a non-trivial performance impact.

Reply:
Good input, thanks!